src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -9385,6 +9407,12 @@ const QList<QgsExpressionFunction *> &QgsExpression::Functions()
           QStringLiteral( "Record and Attributes" )
         );
 
+    // **CRS** functions
+    functions
+        << new QgsStaticExpressionFunction( QStringLiteral( "crs_authid" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "crs" ) ), fcnGetCrsAuthid, QStringLiteral( "CRS" ), QString(), true )
+        << new QgsStaticExpressionFunction( QStringLiteral( "crs" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "definition" ) ), fcnCrs, QStringLiteral( "CRS" ) );
```

Comment:
How about "make_crs", to be inline with eg "make_point"?

```suggestion
        << new QgsStaticExpressionFunction( QStringLiteral( "make_crs" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "definition" ) ), fcnCrs, QStringLiteral( "CRS" ) );
```

Reply:
Renamed to `make_crs`.